olm-catalog/serverless-operator/project.yaml

```diff
@@ -9,7 +9,8 @@ olm:
   channels:
     default: '4.6'
     list:
-      - '4.6'
+      - 4.6
+      - 4.7
```

Comment:
```suggestion
      - '4.6'
      - '4.7'
```
Without quotes they will be read as floats, but they should be strings. I advised adding quotes.